### ISSUE TITLE
Supported Bitnami chart list

### ISSUE THREAD

**Issue by ZCube** [ZCube/bitnami-compat#44]
## Chart List

---

* Chart
  * Image

---

* [x] cert-manager
* [x] common
* [x] consul
* [x] contour
* [x] contour-operator
* [x] discourse
* [x] elasticsearch
* [x] etcd
* [x] external-dns
* [x] fluentd
* [x] kafka
* [x] keycloak
* [x] mariadb
* [x] mariadb-galera
* [x] memcached
* [x] minio
* [x] mongodb
* [x] mongodb-sharded
* [x] mysql
* [x] nginx
* [x] postgresql
* [x] postgresql-ha
* [x] rabbitmq
* [x] rabbitmq-cluster-operator
* [x] redis
* [x] redis-cluster
* [x] sealed-secrets
* [x] solr
* [x] thanos
* [x] tomcat
* [x] wordpress
* [x] zookeeper
* [ ] apache
  ❌  apache
  ✔️  git
  ✔️  apache-exporter
* [ ] gitea
  ❌  gitea
  ✔️  bitnami-shell
* [ ] nginx-ingress-controller
  ❌  nginx-ingress-controller
  ✔️  nginx
* [ ] nginx-intel
  ❌  nginx-intel
  ✔️  git
  ✔️  nginx-exporter
* [ ] wordpress-intel
  ❌  wordpress-intel
  ✔️  bitnami-shell
  ✔️  nginx-exporter
* [ ] airflow
  ✔️  bitnami-shell
  ❌  airflow
  ❌  airflow-scheduler
  ❌  airflow-worker
  ✔️  git
  ❌  airflow-exporter
* [ ] appsmith
  ❌  appsmith
  ✔️  bitnami-shell
* [ ] argo-cd
  ❌  argo-cd
  ❌  dex
  ✔️  bitnami-shell
  ✔️  redis
* [ ] aspnet-core
  ❌  aspnet-core
  ✔️  git
  ❌  dotnet-sdk
* [ ] cassandra
  ❌  cassandra
  ✔️  bitnami-shell
  ❌  cassandra-exporter
* [ ] clickhouse
  ❌  clickhouse
  ✔️  bitnami-shell
* [ ] concourse
  ❌  concourse
  ✔️  bitnami-shell
* [ ] dokuwiki
  ❌  dokuwiki
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] drupal
  ❌  drupal
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] ghost
  ❌  ghost
  ✔️  bitnami-shell
* [ ] grafana
  ❌  grafana
  ❌  grafana-image-renderer
  ✔️  bitnami-shell
* [ ] grafana-loki
  ❌  grafana-loki
  ✔️  nginx
  ❌  promtail
  ✔️  bitnami-shell
  ✔️  memcached
  ✔️  memcached
  ✔️  memcached
  ✔️  memcached
* [ ] grafana-tempo
  ❌  grafana-tempo
  ❌  grafana-tempo-query
  ❌  grafana-tempo-vulture
  ✔️  bitnami-shell
* [ ] harbor
  ✔️  bitnami-shell
  ✔️  nginx
  ❌  harbor-portal
  ❌  harbor-core
  ❌  harbor-jobservice
  ❌  harbor-registry
  ❌  harbor-registryctl
  ❌  chartmuseum
  ❌  harbor-notary-server
  ❌  harbor-notary-signer
  ❌  harbor-adapter-trivy
  ❌  harbor-exporter
  ✔️  postgresql
* [ ] influxdb
  ❌  influxdb
  ✔️  bitnami-shell
  ❌  google-cloud-sdk
  ❌  azure-cli
  ❌  aws-cli
* [ ] jenkins
  ❌  jenkins
  ✔️  bitnami-shell
* [ ] joomla
  ❌  joomla
  ✔️  apache-exporter
* [ ] jupyterhub
  ❌  jupyterhub
  ❌  configurable-http-proxy
  ❌  jupyter-base-notebook
  ✔️  bitnami-shell
* [ ] kibana
  ❌  kibana
  ✔️  bitnami-shell
* [ ] kong
  ❌  kong
  ❌  kong-ingress-controller
  ✔️  postgresql
* [ ] kube-prometheus
  ❌  prometheus-operator
  ❌  prometheus
  ✔️  thanos
  ❌  alertmanager
  ❌  blackbox-exporter
* [ ] kubeapps
  ✔️  nginx
  ❌  kubeapps-dashboard
  ❌  kubeapps-apprepository-controller
  ❌  kubeapps-asset-syncer
  ❌  oauth2-proxy
  ❌  kubeapps-pinniped-proxy
  ❌  kubeapps-apis
* [ ] logstash
  ❌  logstash
  ✔️  bitnami-shell
* [ ] magento
  ❌  magento
  ✔️  mariadb
  ✔️  elasticsearch
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] matomo
  ❌  matomo
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] mediawiki
  ❌  mediawiki
  ✔️  apache-exporter
* [ ] moodle
  ❌  moodle
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] mxnet
  ❌  mxnet
  ✔️  git
  ✔️  bitnami-shell
* [ ] opencart
  ❌  opencart
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] osclass
  ❌  osclass
  ✔️  bitnami-shell
  ✔️  bitnami-shell
  ✔️  apache-exporter
* [ ] owncloud
  ❌  owncloud
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] parse
  ❌  parse
  ❌  parse-dashboard
  ✔️  bitnami-shell
* [ ] phpbb
  ❌  phpbb
  ✔️  bitnami-shell
  ✔️  apache-exporter
* [ ] phpmyadmin
  ❌  phpmyadmin
  ✔️  apache-exporter
* [ ] prestashop
  ❌  prestashop
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] pytorch
  ❌  pytorch
  ✔️  git
  ✔️  bitnami-shell
* [ ] redmine
  ❌  redmine
  ✔️  bitnami-shell
* [ ] sonarqube
  ❌  sonarqube
  ✔️  bitnami-shell
  ✔️  bitnami-shell
  ✔️  jmx-exporter
* [ ] spring-cloud-dataflow
  ❌  spring-cloud-dataflow
  ❌  spring-cloud-dataflow-composed-task-runner
  ❌  spring-cloud-skipper
  ❌  prometheus-rsocket-proxy
  ✔️  kubectl
* [ ] suitecrm
  ❌  suitecrm
  ✔️  bitnami-shell
  ✔️  apache-exporter
  ✔️  bitnami-shell
* [ ] wildfly
  ❌  wildfly
  ✔️  bitnami-shell
* [ ] argo-workflows
  ❌  argo-workflow-cli
  ❌  argo-workflow-controller
  ❌  argo-workflow-exec
* [ ] ejbca
  ❌  ejbca
* [ ] grafana-operator
  ❌  grafana-operator
  ❌  grafana
* [ ] haproxy
  ❌  haproxy
* [ ] haproxy-intel
  ❌  haproxy-intel
* [ ] jasperreports
  ❌  jasperreports
* [ ] kiam
  ❌  kiam
* [ ] kube-state-metrics
  ❌  kube-state-metrics
* [ ] kubernetes-event-exporter
  ❌  kubernetes-event-exporter
* [ ] metallb
  ❌  metallb-controller
  ❌  metallb-speaker
* [ ] metrics-server
  ❌  metrics-server
* [ ] nats
  ❌  nats
  ❌  nats-exporter
* [ ] node-exporter
  ❌  node-exporter
* [ ] oauth2-proxy
  ❌  oauth2-proxy
* [ ] odoo
  ❌  odoo
* [ ] pinniped
  ❌  pinniped
* [ ] schema-registry
  ❌  schema-registry
* [ ] spark
  ❌  spark
* [ ] tensorflow-resnet
  ❌  tensorflow-serving
  ❌  tensorflow-resnet
* [ ] wavefront
  ❌  wavefront-kubernetes-collector
  ❌  wavefront-proxy
* [ ] wavefront-hpa-adapter
  ❌  wavefront-hpa-adapter
* [ ] wavefront-prometheus-storage-adapter
  ❌  wavefront-prometheus-storage-adapter

Generated from
```
go run main.go checkCharts --level=4
```

## Todo

* [ ] CI update this issue.
* [x] #48
  * If there is an official chart, it's a low priority.

**Comment by ZCube** [comment]
I'm closing this issue because I think it's better to keep this as a document in the repository.
